src/components/CourseList.js

```diff
@@ -1,42 +1,23 @@
 import React from 'react';
-
-// The icons for the courses, i.e. all ./*/logo-black.png under lessonSrc, e.g. ./scratch/logo-black.png
-const iconContext = require.context('lessonSrc/', true, /^\.\/[^\/]*\/logo-black\.png/);
-//console.log(iconContext.keys());
+import CourseItem from './CourseItem';
 
 const CourseList = React.createClass({
-  contextTypes: {
-    router: React.PropTypes.object
-  },
-  onClick(path) {
-    const basePath = /^\.(\/[^\/]*)/.exec(path);
-    //console.log(basePath);
-    if (basePath) {
-      this.context.router.push(basePath[1]);
-    }
-  },
+
   render() {
-    const icons = iconContext.keys().map((path, idx) =>{
+    const courses = this.props.courses.map((course, idx) => {
+      if (course.lessons.length == 0) return null;
       return (
-        <div key={idx} style={{display:'inline-block'}}>{icons}
-          <img src={iconContext(path)}
-               onClick={this.onClick.bind(null, path)}
-               style={{width:'80px', margin:'20px'}}/>
-        </div>
+        <CourseItem key={idx} course={course}/>
       );
     });
+
     return (
       <div>
         <h3>Kurs</h3>
-        {icons}
+        {courses}
       </div>
     );
   }
 });
-//
-// <ul>
-//   <li><NavLink to="/" onlyActiveOnIndex>Front Page</NavLink></li>
-//   <li><NavLink to="/scratch">Scratch</NavLink></li>
-//   <li><NavLink to="/python">Python</NavLink></li>
-// </ul>
+
```

Comment:
Jeg ser jo at du bruker material UI her. Tror du at dere kommer til å bruke dette i stedet for bootstrap, og at vi derfor bør disable / fjerne Bootstrap? Foreløpig er det bare bootstrap sin normalizr som er aktivert, så det gjør jo ingenting om den er der, men hvis det er material ui som skal brukes, så vil vi kanskje på sikt fjerne bootstrap?